code/modules/clothing/clothing.dm

```diff
@@ -341,7 +341,6 @@ BLIND     // can't see anything
 	var/can_adjust = 1
 	var/adjusted = 0
 	var/alt_covers_chest = 0 // for adjusted/rolled-down jumpsuits, 0 = exposes chest and arms, 1 = exposes arms only
-	var/suit_color = null

```

Comment:
good ol paprika, adding a var to all jumpsuit to resolve one fucking issue that he created. Good job... All that was needed was two lines in update_inv_w_uniform().